packages/neon_framework/lib/src/blocs/login_check_account.dart

```diff
@@ -1,3 +1,5 @@
+// ignore_for_file: public_member_api_docs
```

Comment:
IIRC the constructors had docs before all these changes. Please restore them or add new ones.

Reply:
Nope that was not the case.

Reply:
Then please add docs.